runtime/doc/filetype.txt

```diff
@@ -542,7 +542,7 @@ completion.
 Command modifiers may be used.  For example, to open in a vertical split: >
 	:vertical Man printf
 
-|:Man| also generates tags for man pages with prefixes matching {name} within
+`:Man` also generates tags for man pages with prefixes matching {name} within
```

Comment:
Is there an advantage to the backquote here?

Reply:
I just read the syntax group, it said `helpHyperTextJump`. While the backquotes produces `helpCommand`. Also, since we are already inside the help docs for `:Man` I thought a "hyper-text jump" was weird.